### ISSUE TITLE
No default config file for lavinmq in the homebrew formula

### ISSUE THREAD

**Issue by snichme** [cloudamqp/lavinmq#823]
**Describe the bug**

No default config in the formula makes it more difficult to get started running LavinMQ.

**How to reproduce**

Install LavinMQ through Homebrew, check results, no config

**Expected behavior**

A basic config file that allows LavinMQ to get up and running with default features.

**Comment by carlhoerberg** [comment]
Absolutely, but shouldnt we have such defaults that a config isnt required? It's a good practice to have a full config with all options, assigned their default value, but commented out, and  a description of each option. Should be able to auto generat from our Config, or hand written, but then we have to remember to update when new options are added or defaults changed.

**Comment by spuun** [comment]
We're now  installing a config, but that config can probably be improved: https://github.com/cloudamqp/homebrew-cloudamqp/pull/21


If we go with #917 we can add more annotations such as "help text" which we could use to generate a config with help for each option.
